src/types/BlockData.ts

```diff
@@ -1,18 +1,16 @@
 export interface BlockData {
-    extraData: string;
+    transactionsCount: number;
+    // ------------------
+    number: string;
```

Comment:
We want to avoid using typescript keywords like `number` as property names for interfaces. rename to `blockNumber` or something similar.

Reply:
discussed